pkg/fluxexec/bootstrap.go

```diff
@@ -180,31 +194,31 @@ func (flux *Flux) bootstrapArgs(opts ...BootstrapOption) []string {
 
 	args := []string{}
 
-	if c.authorEmail != "" {
+	if c.authorEmail != "" && !reflect.DeepEqual(c.authorEmail, defaultBootstrapOptions.authorEmail) {
```

Comment:
:grey_question:  Could you explain why you did this (e.g. "all the DeepEquals", I just picked one for the comment) change?

Reply:
The risk is that the default we think flux has diverges from the actual flux default, and then if you try to set "our default" we'd just remove it and flux would then set "the new flux default".

Though I suppose that's entirely hypothetical.

Reply:
yes, the consequence is that we have to revisit this package every time Flux releasing a new version. 
It is the case for HC's tfexec as well.

I'll document how to do so, systematically.